scripts/upload_code.py

```
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
+"""
+Deployment script using cargo contract
+"""
+
+import os
+import sys
+import json
+import subprocess
+import hashlib
+from datetime import datetime
+from typing import Dict, Optional, Tuple
```

Flake8 (7.2.0)

[error] 'typing.Dict' imported but unused

(F401)